Package.swift

```diff
@@ -13,7 +13,11 @@ let package = Package(
   ],
   products: [
     .library(name: "IssueReporting", targets: ["IssueReporting"]),
-    .library(name: "IssueReportingTestSupport", targets: ["IssueReportingTestSupport"]),
+    .library(
+      name: "IssueReportingTestSupport",
+      type: .dynamic,
```

Comment:
We need to force this to be dynamic for Linux/Windows to be able to dynamically load its symbols.